Sources/Sentry/SentryHybridSKdsOnly.m

```diff
@@ -41,4 +41,9 @@ + (nullable SentryEnvelope *)envelopeWithData:(NSData *)data
     return [self.debugImageProvider getDebugImages];
 }
 
++ (nullable SentryAppStartMeasurement *)getAppStartMeasurement
```

Comment:
Just tried this out via putting the commit hash in the podfile on RN.

So turns out this wouldn't be available until `UIWindowDidBecomeVisibleNotification` is posted per @philipphofmann, this means a callback would be required as RN wouldn't know when to call this otherwise.